docs/spec/blockchain/blockchain.md

```diff
@@ -107,6 +108,29 @@ There are two types of votes:
 a *prevote* has `vote.Type == 1` and
 a *precommit* has `vote.Type == 2`.
 
+## Commit
+
+A Commit contains enough data to verify that a BlockID was committed to at a certain
+height and round by a sufficient set of validators.
+
+```go
+type Commit struct {
+    HeightNum  int64
```

Comment:
I would agree with using the standard variable names, but we need the `Height()` getters to implement the `VoteSetReader` interface so that `Commit`s can have votes pulled from them for gossiping via the same process as in progress vote sets. I would be open to alternative names tho, as `HeightNum` is a bit awkward.

Reply:
What if we make them private and use `NewCommit` (or create additional constructors) to create `Commit`

Reply:
In the spec we don't have to worry about collisions with method names, we can just call it Height.

Reply:
I think it should be Height, and the method can be GetHeight().  Adding "Num" due to a method name conflict is surprising.  Adding a "Get" prefix is already convention.